### ISSUE TITLE
All exceptions are showing up in Honeybadger as the same error

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/abid#156]
https://app.honeybadger.io/projects/87494/faults/102500208/01HRNCKY8TYT53XDH90J8KQX4X?q=occurred.after%3A%222+weeks+ago%22

## Success Criteria

Better errors.

## Step One

Google the error we're seeing.

We should be able to fix this by upgrading Rails to 7.1.3.2

**Comment by tpendragon** [comment]
The Honeybadger version we had isn't working with Rails 7.1.

However, upgrading Honeybadger makes errors not send either - because of https://github.com/rails/rails/pull/51050, which is not yet in a release.

**Comment by tpendragon** [comment]
I don't think we can fix this until that Rails patch above is in a release.

**Comment by hackartisan** [comment]
The PR linked above, that we're waiting on release of, was merged on Feb 12 with a comment on that same date that it was backported to 7.1 and 7.0.

abid is on 7.1. I checked the Rails 7.1-stable branch history and this commit is [3 pages in](https://github.com/rails/rails/commits/7-1-stable?after=c3c164ca12410983e07537531844c4255450ba9b+69) with sha cca9eef. Navigating "back" (i.e. forward in time) from that page you see commits about preparing for 7.1.3.1 and 7.1.3.2. So even though the [7.1.3.2 release(https://rubyonrails.org/2024/2/21/Rails-Versions-6-1-7-7-7-0-8-1-and-7-1-3-2-have-been-released) announcement doesn't mention this change, I'm pretty sure it's in there. So I think this ticket is unblocked.
